src/test/java/org/assertj/core/api/duration/DurationAssert_hasDays_Test.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.api.duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.error.ShouldHaveDuration.shouldHaveDays;
+import static org.assertj.core.util.AssertionsUtil.assertThatAssertionErrorIsThrownBy;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+import java.time.Duration;
+
+import org.assertj.core.api.AbstractDurationAssert;
+import org.assertj.core.api.BaseTest;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Filip Hrisafov
+ */
+@DisplayName("DurationAssert hasDays")
+class DurationAssert_hasDays_Test extends BaseTest {
+
+  @Test
+  void should_pass_if_duration_has_matching_days() {
+    // GIVEN
+    Duration duration = Duration.ofDays(4L);
+    //WHEN
+    AbstractDurationAssert<?> durationAssert = assertThat(duration);
+    //THEN
+    assertThat(duration).hasDays(4L);
```

Comment:
I think this test could be simplified to:
```java
    // GIVEN
    Duration duration = Duration.ofDays(4L);
    // WHEN
    assertThat(duration).hasDays(4L);
```

Reply:
Honestly I liked the previous `assertThat(Duration.ofDays(4L)).hasDays(4L)`. I had a look at in one of the recent commits and it was done in the way that I adapted it. Feel free to change it if you want.